src/views/studentregistration/studentregistration.jsx

```diff
@@ -126,7 +126,7 @@ class StudentRegistration extends React.Component {
                                 id: 'registration.studentUsernameStepTooltip'
                             })}
                             usernameHelp={this.props.intl.formatMessage({
-                                id: 'registration.studentUsernameFieldHelpText'
+                                id: 'registration.studentUsernameSuggestion'
```

Comment:
Only thought is that now `studentUsernameFieldHelpText` is an orphan. Should we remove it? Or just replace the content and use it instead of `studentUsernameSuggestion`?

Reply:
Good point. I removed the unused message.